packages/server-wallet/src/wallet/index.ts

```diff
@@ -80,6 +85,35 @@ export type WalletInterface = {
 };
 
 export class Wallet implements WalletInterface {
+  public async syncChannel({channelId}: SyncChannelParams): SingleChannelResult {
+    return Store.lockApp(
```

Comment:
Do we need to lock the app here? It's only reading data, so it looks like it might be fine not to?

Reply:
We do not — this is a relic of the previous API. Let me remove...